modules/standard/LinkedLists.chpl

```diff
@@ -26,7 +26,7 @@
       This module is expected to change in the future.
  */
 module LinkedLists {
-  private use SysBasic;
+  use SysBasic;
```

Comment:
I don't see why this line is needed

Reply:
I think you're saying "Why is LinkedLists using SysBasic" rather than "why did you remove `private`", is that right?  In this PR, I wasn't trying to reduce the number of `use`s that each module contains (though I'd like to do more of that in later passes), but just trying to reduce the number of symbols that are improperly made available to user code (so didn't try to reason about whether each `use` was necessary).

Reply:
OK